### PR TITLE
Jk/bugfix/landing query

### LINKED ISSUE #58: Local Storage Refresh Delay
Logging out, refreshing the login page, and logging in as a new user will display old logged in data.
Need to implement an await for local storage fetch.

### PR DESCRIPTION
**Pull request by JeffKersting**
#### Purpose:
 > Check all that apply.
- [ ] Feature
- [ ] Refactor
- [x] Bug Fix 
- [ ] Clean Up

### What does this PR do?
 > Clearly and concisely describe changes made. 

Sets the userID hook in App.js directly on Login page once retrieved from storage.
Clears apollo cache on logout (Header.js)
Passes isLoggedIn hook from App to Home (Home already written to take data)

Issues closed #58 

N/A

App: line 39 & 48
Login: line 17

Logout, login as new user, verify that you do not see previous user data and new data loads properly.

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -35,14 +35,17 @@ const App = () => {
       <Route
         exact
         path='/'
-        render={() => <Home />}
+        render={() => <Home
+            isLoggedIn={isLoggedIn}
```
Comment (gaj23) [inline src/components/App/App.js:39]: Glad to get this little thing put to rest!

File: src/components/Header/Header.js [hunk 2]
```diff
@@ -23,6 +25,7 @@ const Header = ({ isLoggedIn, setIsLoggedIn }) => {
 
   const clearLocalStorage = () => {
     localStorage.clear()
+    client.resetStore()
```
Comment (gaj23) [inline src/components/Header/Header.js:28]: Grateful the problem of the cache overwriting our current session was a fairly easy fix!

File: src/components/Login/Login.js [hunk 3]
```diff
@@ -5,18 +5,20 @@ import PropTypes from 'prop-types';
...
   const [email, setEmail] = useState('');
   const [password, setPassword] = useState('');
-  // ensure error component is added once connected to the backend
 
-  // const { error, loading, data } = useQuery(GET_USER)
-  const [signinUser] = useMutation(SIGNIN_USER, {onCompleted: data => localStorage.setItem('userData', JSON.stringify(data.signinUser.user.id))});
-  
+  const [signinUser] = useMutation(SIGNIN_USER, {
+    onCompleted: data => {
+      setID(parseInt(data.signinUser.user.id))
```
Comment (gaj23) [inline src/components/Login/Login.js:17]: To clarify, this just double checks to make sure that the ID is correct all across the board.

**Review by gaj23 — APPROVED** [review]
Feels good to have things a little tighter on our end. 
Nice work everyone!

File: src/components/Header/Header.js (hunk 2 above)
Comment (kelsiebesingeryeh) [reply src/components/Header/Header.js:28]: yay

**Review by kelsiebesingeryeh — APPROVED** [review]

**Pull request merged**